website/docs/docs/quickstarts/overview.md

```diff
@@ -43,9 +43,15 @@ Before you begin, you will need:
     icon="snowflake"/>
 
 <Card
-    title="Quickstart for dbt Core"
+    title="Quickstart for dbt Core using GitHub Codespaces"
+    body="Learn how to quickly start running dbt commands in a GitHub codespace development environment with just a couple clicks."
+    link="/docs/quickstarts/dbt-core/codespace"
+    icon="dbt-core"/>
```

Comment:
@jtcohen6 / @gwenwindflower : what logo/icon should we use for this new Card? should we do GH + DuckDB?

Reply:
This makes me think that, instead of the `dbt Core` logo in both places, we'd want to use:
- GitHub logo (for Codespace)
- An icon that signifies "terminal" / "CLI" - like [these](https://www.flaticon.com/free-icons/command-line)

Part of the motivation there is also recognizing that, if folks pick one of the dbt Cloud quickstarts, they're not _not_ using `dbt Core` (running in Cloud). It's all dbt.